README.md

```diff
@@ -34,19 +34,19 @@ Tested, verified, and supported by observIQ
 
 To install using the installation script, you may run:
 ```sh
-sudo sh -c "$(curl -fsSlL https://github.com/observiq/observiq-otel-collector/releases/latest/download/install_unix.sh)" install_unix.sh
+sudo sh -c "$(curl -fsSlL https://github.com/observiq/bindplane-agent/releases/latest/download/install_unix.sh)" install_unix.sh
 ```
 
 To install directly with the appropriate package manager, see [installing on Linux](/docs/installation-linux.md).
 
 #### Windows
 
-To install the collector on Windows run the Powershell command below to install the MSI with no UI.
+To install the BindPlane Agent on Windows run the Powershell command below to install the MSI with no UI.
 ```pwsh
-msiexec /i "https://github.com/observIQ/observiq-otel-collector/releases/latest/download/observiq-otel-collector.msi" /quiet
+msiexec /i "https://github.com/observIQ/bindplane-agent/releases/latest/download/observiq-otel-collector.msi" /quiet
 ```
 
-Alternately, for an interactive installation [download the latest MSI](https://github.com/observIQ/observiq-otel-collector/releases/latest).
+Alternately, for an interactive installation [download the latest MSI](https://github.com/observIQ/bindplane-agent/releases/latest).
 
 After downloading the MSI, simply double click it to open the installation wizard. Follow the instructions to configure and install the collector.
```

Comment:
agent

Reply:
Saw a few more of these and replaced them.